src/storage/heap_file.c

```diff
@@ -14530,9 +14531,13 @@ heap_dump_heap_file (THREAD_ENTRY * thread_p, FILE * fp, bool dump_records, cons
   status = xlocator_find_class_oid (thread_p, class_name, &class_oid, S_LOCK);
   if (status != LC_CLASSNAME_EXIST)
     {
+      fprintf (stderr, msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_DIAGDB, DIAGDB_MSG_UNKNOWN_CLASS),
```

Comment:
I think it makes sense for utility error messages to be handled within the utility itself. 
This function simply return an error code, and the utility could use that error code to output the error message.
The message should not only be printed to stderr but also logged in the utility error file. What do you think?

Reply:
I checked whether optimizedb writes a log in the utility error file when a given class does not exist. I found that optimizedb does not log in this situation. I could be mistaken, but if optimizedb indeed behaves this way, I believe diagdb should also avoid logging in the utility error file for consistency.

Reply:
Please, see why and how we use `PRINT_AND_LOG_ERR_MSG` function in `diagdb`. 
There is no need to follow the implementation of optimizedb. It is just logging the necessary situations. 

And the server and utilities should be separated and display error messages accordingly.